### ISSUE TITLE
FAQs

### ISSUE THREAD

**Issue by InruptKelly** [solid/solid.github.io#105]
Now that https://github.com/solid/solid.github.io/blob/master/pages/faqs.md will soon form the basis of an important section of Solid's most public-facing website, I suggest we do a thorough review and update of the information there. 

I suspect the answers to some of these questions are outdated from a technical perspective. And I think overall the list is somewhat overwhelming to newcomers. Perhaps it would be best to start from scratch and try to limit ourselves to the TOP 10 most frequently asked questions. That's a much more manageable way for newcomers to start learning about Solid. Or at the least, I think we should organize the questions into general themes/topics so people can easily identify what they're interested in.

I'm happy to volunteer some time to help update the FAQs.

**Comment by Vinnl** [comment]
A review and update would certainly be a good idea. As for limiting them, however, one piece of feedback we got remarkably often during user testing is that people liked the length of the FAQs page, with comments like "I had a lot of questions, so I'll definitely look through this thoroughly later on" and "this looks like you've thought of most things".

That said, another piece of feedback we got a lot is that people would like a table of contents there, so that's certainly something we should be picking up. Additionally, a pattern I've seen elsewhere is to split the FAQ page up into two sections: "Top 10 most frequently asked questions" and "Other Q&As". Although it's anecdotal, I found that helpful to make it feel less overwhelming, so that might be something for us to consider as well.
